src/features/pass-survey/ui/ActivityItem.tsx

```diff
@@ -17,6 +17,7 @@ import {
   StackedSlider,
 } from '@app/shared/ui';
 import { HandlersContext } from '@app/shared/ui';
+import TextParagraph from '@app/shared/ui/survey/TextParagraph';
```

Comment:
Please change the import to:

```suggestion
import { HandlersContext , TextParagraph } from '@app/shared/ui';
```

Reply:
sorry, it was automatic import, I will fix it.